ODIN_II/SRC/netlist_cleanup.cpp

```diff
@@ -237,11 +242,81 @@ void calculate_addsub_statistics(node_list_t* addsub) {
     /* Calculate the geometric mean carry chain length */
     geomean_addsub_length = exp(sum_of_addsub_logs / total_addsub_chain_count);
 }
+void count_node_type(nnode_t* node) {
+    switch (node->type) {
+        // fallthrough
+        case LOGICAL_OR:
+        case LOGICAL_AND:
+        case LOGICAL_NOR:
+        case LOGICAL_NAND:
+        case LOGICAL_XOR:
+        case LOGICAL_XNOR:
+        case LOGICAL_NOT:
+        case LOGICAL_EQUAL:
+        case NOT_EQUAL:
+        case BITWISE_NOT:
+        case BUF_NODE:
+        case LT:
+        case GT:
+        case FF_NODE:
+        case MEMORY:
+        case MUX_2:
```

Comment:
please count smux and mux together and show both of them as mux, like [here](https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/e8b1b04c579feb0bc881fc95d2f42a42b100f591/ODIN_II/SRC/netlist_statistic.cpp#L163)